src/actions/actionCreators.js

```diff
@@ -8,3 +8,44 @@ import * as types from './actionTypes';
   })
 
 */
+export const login = (username, password) => (dispatch) => {
+  // replace the url to login route
+  // console.log(username, password);
+  fetch('https://jsonplaceholder.typicode.com/posts', {
+    method: 'POST',
+    body: JSON.stringify({
+      username: 'davidkim',
+      email: 'davidkim@email.com',
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then(response => response.json())
+    .then((json) => {
+      // simulate user authentication
+      const isAuthenticated = json.email === 'davidkim@email.com';
+      dispatch({ type: types.SIGN_UP, payload: { isLoggedIn: isAuthenticated } });
+    });
+};
+
+export const signup = (username, password, firstname, lastname, email) => (dispatch) => {
+  // console.log(username, password, firstname, lastname, email);
```

Comment:
Hey David can you remove this console log?

Reply:
done